app-scientific/sagemath/autobuild/beyond

```diff
@@ -0,0 +1,2 @@
+export SAGE_LOCAL="$PKGDIR"/usr/lib/sagemath/local
+chmod +x "$SAGE_LOCAL"/lib/*.so*
```

Comment:
Just use that path, there is no real need for that variable...

Also, please add `abinfo` and verbosity per the [Styling Manual](https://wiki.aosc.io/zh/developer/packaging/package-styling-manual/):

```
abinfo "Setting executable bits on all shared libraries ...
chmod -v +x "$PKGDIR"/usr/lib/sagemath/local/lib/*.so*
```